gems/sorbet/test/snapshot/run_one_bazel.sh

```diff
@@ -93,6 +93,17 @@ srb="${repo_root}/gems/sorbet/bin/srb"
 info "├─ sorbet:           $SRB_SORBET_EXE"
 info "├─ sorbet --version: $("$SRB_SORBET_EXE" --version)"
 
+SORBET_TYPED_REV="$(rlocation com_stripe_ruby_typer/gems/sorbet/test/snapshot/sorbet-typed.rev)"
+SRB_SORBET_TYPED_REVISION="$(<"$SORBET_TYPED_REV")"
```

Comment:
TIL

Reply:
Yeah, it's a little painful to refer to runfiles in an sh_binary :(